### PR TITLE
add UNICODE funciton #13

### PR DESCRIPTION
**Pull request by tbro**

### CODE REVIEW

File: crates/gitql-ast/src/function.rs [hunk 1]
```diff
@@ -371,6 +380,15 @@ fn text_translate(inputs: Vec<Value>) -> Value {
...
 }
+fn text_unicode(inputs: Vec<Value>) -> Value {
+    let n = if let Some(c) = inputs[0].as_text().chars().next() {
+        c as u32
+    } else {
+        return Value::Integer(0);
+    };
+
+    return Value::Integer(n.into());
+}
```
Comment (AmrDeveloper) [inline crates/gitql-ast/src/function.rs:391]: Your code looks good but we can make it more similar to read like below, think of it as return result or default value 

```rs
if let Some(c) = inputs[0].as_text().chars().next() {
   return Value::Integer((c as u32).into());
}
return Value::Integer(0);
```

File: docs/function/functions.md [hunk 2]
```diff
@@ -41,6 +41,7 @@ SELECT name, SUBSTRING(name, 1, 5) AS extract FROM commits
 SELECT STUFF("GQL tutorial!", 13, 1, " is fun!")
 SELECT RIGHT("AmrDeveloper", 3) AS extract
 SELECT TRANSLATE("Amr[Dev]{eloper}", "[]{}", "()()")
+SELECT UNICODE("AmrDeveloper")
```
Comment (AmrDeveloper) [inline docs/function/functions.md:44]: Missing documentations for `UNICODE` function on String functions table above

**Review by AmrDeveloper — CHANGES_REQUESTED** [review]

File: crates/gitql-ast/src/function.rs (hunk 1 above)
Comment (tbro) [reply crates/gitql-ast/src/function.rs]: thanks for the feedback. Fixed!

File: docs/function/functions.md (hunk 2 above)
Comment (tbro) [reply docs/function/functions.md:45]: Sorry about that. Fixed!

File: crates/gitql-ast/src/function.rs [hunk 3]
```diff
@@ -371,6 +380,12 @@ fn text_translate(inputs: Vec<Value>) -> Value {
 
     return Value::Text(text);
 }
+fn text_unicode(inputs: Vec<Value>) -> Value {
```
Comment (AmrDeveloper) [inline crates/gitql-ast/src/function.rs:383]: @tbro Please fix conflicts from the previous PR and add line between the two functions

**Review by AmrDeveloper — CHANGES_REQUESTED** [review]

File: docs/function/functions.md (hunk 2 above)
Comment (AmrDeveloper) [reply docs/function/functions.md:45]: No problem, thanks

File: crates/gitql-ast/src/function.rs (hunk 1 above)
Comment (AmrDeveloper) [reply crates/gitql-ast/src/function.rs]: Good thanks

File: crates/gitql-ast/src/function.rs [hunk 4]
```diff
@@ -189,6 +190,13 @@ lazy_static! {
                 result: DataType::Text
              },
         );
+        map.insert(
+            "unicode",
+            Prototype {
+                parameters: vec![DataType::Text],
+                result: DataType::Text
```
Comment (AmrDeveloper) [inline crates/gitql-ast/src/function.rs:197]: UNICODE prototype should return an Integer not Text

**Review by AmrDeveloper — CHANGES_REQUESTED** [review]

File: docs/function/functions.md [hunk 5]
```diff
@@ -25,6 +25,7 @@ note that all functions names are case-insensitive.
 | TRANSLATE  | Text, Text, Text,          | Text   | Returns the string from the first argument after the characters specified in the second argument are translated into the characters specified in the third argument. |
 | SOUNDEX    | Text                       | Text   | Returns a four-character code to evaluate the similarity of two expressions. |
 | CONCAT     | Text, Text                 | Text   | Adds two or more strings together.                                          |
+| UNICODE    | Text                       | Number | Return an integer value (the Unicode value), for the first character of the input expression.                                                                        |
 ### String functions samples
```
Comment (AmrDeveloper) [inline docs/function/functions.md:29]: Please add new line between functions table and the title `String functions samples`

**Review by AmrDeveloper — CHANGES_REQUESTED** [review]

**Comment by AmrDeveloper** [comment]
Thank you for your work @tbro,

**Review by AmrDeveloper — APPROVED** [review]

**Pull request merged**